src/main/java/uk/gov/hmcts/ccd/data/casedetails/CaseDetailsEntity.java

```diff
@@ -77,8 +77,8 @@
     public static final String DATA_COL = "data";
     public static final String DATA_CLASSIFICATION_COL = "data_classification";
     public static final String SUPPLEMENTARY_DATA_COL = "supplementary_data";
-
-
+    public static final String LOCATION = "data.caseManagementLocation.BaseLocationId";
+    public static final String REGION = "data.caseManagementLocation.RegionId";
```

Comment:
```suggestion
    public static final String LOCATION = "data.caseManagementLocation.BaseLocationId.keyword";
    public static final String REGION = "data.caseManagementLocation.RegionId.keyword";
```
These should be `keyword` too, right?